### PR TITLE
Pr view

### PR DESCRIPTION
**Pull request by DavidJones1102**
Show comments to pr,
add clicable pr link

### CODE REVIEW

**Comment by M0rgho** [comment]
good PR, hovever I'm requesting a couple changes:
    - boundaries between commits are not clear, split them with maybe a horizontal line, or some other way to distinguish them
    - some comments can be pretty long, this can cause annoying scrolling - create a character limit for comment and replace the overflow with "..." 
    - add some color to distinguish open and closed PRs, don't sort them I'll do it later

**Review by M0rgho — CHANGES_REQUESTED** [review]
missclick, changes requested in previous comment

**Comment by DavidJones1102** [comment]
> good PR, hovever I'm requesting a couple changes: - boundaries between commits are not clear, split them with maybe a horizontal line, or some other way to distinguish them - some comments can be pretty long, this can cause annoying scrolling - create a character limit for comment and replace the overflow with "..." - add some color to distinguish open and closed PRs, don't sort them I'll do it later

I couldn't color open pr because pr.isMerged() is always false. It needs to be repaired in future.
I applied other suggestions

File: src/main/java/com/io/gittracker/controllers/RepositoryPresenter.java [hunk 1]
```diff
@@ -74,7 +74,37 @@ private void updatePullRequestsView(ObservableList<PullRequest> newPullRequests)
...
+                            prVbox.getChildren().add(prLabel);
+                            prVbox.getChildren()
+                                    .addAll(pr.getComments().stream()
+                                            .map(prComment -> {
+                                                Label label = new Label(prComment.username() + " commented: "
+                                                        + (prComment.body().length() > 200
+                                                                ? prComment
+                                                                                .body()
+                                                                                .substring(0, 197) + "[...]"
+                                                                : prComment.body()));
```
Comment (DavidJones1102) [inline src/main/java/com/io/gittracker/controllers/RepositoryPresenter.java:100]: spotlessApply made this weird formatting

**Review by M0rgho — APPROVED** [review]
good

**Pull request merged**